module-apps/application-phonebook/widgets/PhonebookItem.cpp

```diff
@@ -65,10 +64,6 @@ namespace gui
     void PhonebookItem::markFavourite(bool val)
```

Comment:
Does markFavourite has any role after these changes?

Reply:
Yes, but actually it is used only once and there is no need to keep it as a function anymore.
Changed